plugin/adsk/plugin/plugin.cpp

```diff
@@ -152,7 +153,8 @@ MStatus initializePlugin(MObject obj)
     g_Transform3dHandler = runTimeMgr.transform3dHandler(usdRtid);
 	auto matrixHandler = MAYAUSD_NS::ufe::UsdTransform3dMatrixOpHandler::create(
 		g_Transform3dHandler);
-	runTimeMgr.setTransform3dHandler(usdRtid, matrixHandler);
+	auto mayaStackHandler = MAYAUSD_NS::ufe::UsdTransform3dMayaXformStackHandler::create(matrixHandler);
```

Comment:
Add the new Maya transform stack Transform3dHandler to the head of the chain of responsibility, which now looks like:

Maya transform stack --> Matrix op transform (stack) --> USD common transform API fallback.

Each Transform3dHandler will be tried in sequence.  If it fails to match the transform op stack, it doesn't understand it, and passes it on to the next handler in the chain.